src/options/theory_options.toml

```diff
@@ -32,6 +32,24 @@ name   = "Theory Layer"
   default    = "true"
   help       = "condense values for functions in models rather than explicitly representing them"
 
+[[option]]
+  name       = "defaultFunctionValueMode"
+  category   = "expert"
+  long       = "default-function-value-mode=MODE"
+  type       = "DefaultFunctionValueMode"
+  default    = "FIRST"
+  help       = "mode for choosing default values for functions"
+  help_mode  = "Defines mode for choosing default values for functions."
+[[option.mode.FIRST]]
+  name = "first"
+  help = "The default value is equal to the value of the first application found of that function when applicable, or the first enumerated term otherwise."
+[[option.mode.FIRST_ENUM]]
+  name = "first-enum"
+  help = "The default value is equal to the first enumerated value of its range type."
```

Comment:
Same as 2 above.

Reply:
For 2, the enumeration is based on cvc5's internal enumeration of values.